forem/forem
# Revert "Run `yarn upgrade` (#14474)"

This reverts commit 82c24417e0007cd19c9eafa6309e36905acdbfd8.



## What type of PR is this? (check all applicable)

- Revert of a dependency change

## Description

After merging #14474 our buildkite builds were erroring with:

```
error jest-regex-util@27.0.6: The engine "node" is incompatible with this module. Expected version "^10.13.0 || ^12.13.0 || ^14.15.0 || >=15.0.0". Got "14.14.0"
error Found incompatible module.
```

This is due to the known issues with the node version in buildkite, and until we're able to bump this to latest we might have to wait on this changes.

## Related Tickets & Documents

#14474

## QA Instructions, Screenshots, Recordings

- Build should pass
- If you're also using node 14.14 locally, you should be able to run `yarn install` successfully

### UI accessibility concerns?

N/A

## Added/updated tests?


## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

      [Admin Guide](https://admin.forem.com/), or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
      or in a [forem.dev](http://forem.dev) post
      replace this line with details on why this change doesn't need to be
      shared_
